geps/gep-1742.md

```diff
@@ -301,8 +301,87 @@ Could not find any HTTP specific timeouts. PRs welcomed. 😊
 
 ## API
 
-TBD.
+From the above analysis, it appears that most implementations are capable of supporting
+the configuration of simple downstream request timeouts on HTTPRoute rules. This is a
+relatively small addition (one new field) that would benefit many users.
+
+Because request timeouts are very much related to retry attempts, we also propose
+addressing issue: [#1731](https://github.com/kubernetes-sigs/gateway-api/issues/1731)
+by adding another field for retry conifguration in an HTTPRoute rule. This would also
+benefit many users and seems to be implementable by many (most?) Gateway implementations.
+
+### GO
+
+```go
+type HTTPRouteRule struct {
+	// Timeout for HTTP requests, disabled by default.
+	Timeout *time.Duration `json:"timeout,omitempty"`
```

Comment:
Can we expand the specification for this a bit? Given the GEP already documents probably 20 types of "timeout" in https://gateway-api.sigs.k8s.io/geps/gep-1742/?h=1742#flow-diagrams-with-available-timeouts, having just a single field as "timeout" with no further explanation seems a bit ambiguous.

Reply:
@arkodg Good question. I think it depends on whether implementations support configuring other kinds of timeouts at the HTTPRoute level. Based on the diagrams in the previous sections, I think Envoy does not, but unclear about the others.

@youngnick do you have more insight on this?

Reply:
@kflynn This suggestion sounds quite good to me. The reason that I dragged retries into this GEP in the first place is because perTryTimeout was so tightly coupled to retries, but moving it out and generalizing it like you suggest allows for better separation of concerns and to only focus on Timeouts in this GEP. I'm happy to make this change but will give others some time to comment first.